core/handlers/v2/hoverfly_pac_handler.go

```diff
@@ -43,8 +43,8 @@ func (this *HoverflyPACHandler) Get(w http.ResponseWriter, req *http.Request, ne
 	if pacFile == nil {
 		handlers.WriteErrorResponse(w, "Not found", 404)
 	}
-	handlers.WriteResponse(w, pacFile)
-	w.Header().Set("Content-Type", "application/x-ns-proxy-autoconfig")
```

Comment:
I thought this line should have set the correct type?

Reply:
you cannot set headers after response is already written. it's written in `handlers.WriteResponse`

Reply:
makes sense 👍